lib/discordrb/data/user.rb

```diff
@@ -166,7 +166,18 @@ def webhook?
       @discriminator == Message::ZERO_DISCRIM
     end
 
-    %i[offline idle online].each do |e|
+    # @!group Statuses
```

Comment:
I'm not a fan of the group here personally, but if we keep it the syntax needs a fixup
```suggestion
    # @!group Statuses

```